src/o365/spo/commands/theme/theme-apply.ts

```diff
@@ -106,10 +160,54 @@ class SpoThemeApplyCommand extends SpoCommand {
         return isValidSharePointUrl;
       }
 
+      if(args.options.sharePointTheme && !(args.options.name in SharePointThemes)){
+        return 'Please check if the theme name is entered correctly.'
```

Comment:
We should list the name of themes we support in the CLI somewhere in the command. Otherwise, this error by itself leaves the user clueless.